smdebug/mxnet/collection.py

```diff
@@ -21,6 +22,8 @@ def __init__(self, create_default=True):
             self._register_default_collections()
 
     def _register_default_collections(self):
+        for c in DEFAULT_MXNET_COLLECTIONS:
+            self.create_collection(c)
```

Comment:
do we need these loops in all _register_default_collections()? collectionmanager.get() creates the collection (if not available) by default. Collection.ALL and DEFAULT are created in core/hook.py and the framework specific _register_default_collections() create their respective default collections.

Reply:
This is redundant but we just want to enforce this style where we separate the creation step from the get call